src/Version.php

```diff
@@ -32,11 +32,56 @@ public static function isStable()
     }
 
     /**
-     * Compares a version to Bolt's version.
+     * Compares a semantic version against Bolt's version, given a specified
+     * comparison operator.
      *
-     * Note: Be sure to include the `.z` number in the version given, as omitting it can give inconsistent results.
-     * For example, if the current version is `3.3.0`, `compare('3.3', '>=')` returns false. In reality you want
-     * the check to return true for this case, and `compare('3.3.0', '>=')` _does_ return true.
+     * @see http://semver.org/ For an exmplaination on semantic versioning.
```

Comment:
IntelliJ/PHPStorm doesn't parse this inline, and thus everything below is left out.

You could do an inline tag:
```
* {@see http://semver.org/ See SemVer} for an explanation on semantic versioning.
```

Or no tag:
```
* See http://semver.org/ for an explanation on semantic versioning.
```

Or put it at the bottom of the phpdoc. 

Or leave it out. 

Personally, I think not even mentioning semver here is better, and just link to [version_compare](http://php.net/manual/en/function.version-compare.php)

----

Also "exmplaination" is misspelled.